include/vsg/vk/Queue.h

```diff
@@ -12,6 +12,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 </editor-fold> */
 
+#include <vsg/core/Inherit.h>
 #include <vsg/core/Object.h>
```

Comment:
The Object.h include is now redundant as it Inherit.h includes it.  I'll merge this PR and remove remove the Object.h header.

Reply:
I have merged this PR and removed the no longer required include: 51cff49917cdaa1b58ab8dd8001a56d15d5a22d8